helm_deploy/values-dev.yaml

```diff
@@ -12,10 +12,24 @@ generic-service:
     ASSESSMENT_ENDPOINT_URL: https://offender-dev.aks-dev-1.studio-hosting.service.justice.gov.uk
     HMPPS_TIER_ENDPOINT_URL: https://hmpps-tier-dev.hmpps.service.justice.gov.uk
     TIER_TO_DELIUS_ENDPOINT_URL: https://tier-to-delius-dev.hmpps.service.justice.gov.uk
+    HMPPS_SQS_USE_WEB_TOKEN: "true"
   allowlist:
     dxw-vpn: "54.76.254.148/32"
     delius-test-1: "35.176.126.163/32"
     delius-test-2: "35.178.162.73/32"
     delius-test-3: "52.56.195.113/32"
   scheduledDowntime:
-    enabled: true
\ No newline at end of file
+    enabled: true
+  namespace_secrets:
```

Comment:
The namespace secrets will be put back into values.yaml when preprod and prod have been done